be/src/olap/base_tablet.cpp

```diff
@@ -909,7 +908,7 @@ Status BaseTablet::fetch_value_through_row_column(RowsetSharedPtr input_rowset,
         serdes[i] = type->get_serde();
     }
     vectorized::JsonbSerializeUtil::jsonb_to_block(serdes, *string_column, col_uid_to_idx, block,
-                                                   default_values);
+                                                   default_values, {});
```

Comment:
Do you mean use all columns from row column? Can it be optimized to only copy needed columns?